mlptrain/sampling/reaction_coord.py

```diff
@@ -82,6 +82,12 @@ def __init__(self, *args):
 
             self.atom_pair_list.append(tuple(arg))
 
+        atom_idxs = [idx for pair in self.atom_pair_list for idx in pair]
+
+        if len(set(atom_idxs)) != len(atom_idxs):
+            raise ValueError('All atoms in reaction coordinate must be '
```

Comment:
not sure I understand this. ζ = r(0–1) - r(2–0) for an SN2 wouldn't be allowed?!